deployments/fuji/usdc/roots.json

```diff
@@ -1,6 +1,3 @@
 {
-    "comet": "0x59BF4753899C20EA152dEefc6f6A14B2a5CC3021",
-    "configurator": "0x8c083632099CBA949EA61A3044DB1B5A27818b20",
-    "rewards": "0x7CA364f9C4257FE2E22d503dD0E3f1c1Db41591d",
-    "fauceteer": "0x45D3465046B72D319ef0090b431678b160B1e628"
+    "telepathyRouter": "0x75d02F008ce008bCC6307784135760795726F281"
```

Comment:
You can actually remove this from roots, I think it'll get re-written in here when the deploy script is run.